Podfile

```diff
@@ -306,25 +255,32 @@ end
 # Make all pods that are not shared across multiple targets into static frameworks by overriding the static_framework? function to return true
 # Linking the shared frameworks statically would lead to duplicate symbols
 # A future version of CocoaPods may make this easier to do. See https://github.com/CocoaPods/CocoaPods/issues/7428
-shared_targets = ['WordPressFlux']
+shared_targets = %w[WordPressFlux]
+dyanmic_framework_pods = %w[WordPressFlux WordPressShared WordPressUI SVProgressHUD Gridicons NSURL+IDN wpxmlrpc NSObject-SafeExpectations]
+# Statically linking Sentry results in a conflict with `NSDictionary.objectAtKeyPath`, but dynamically
+# linking it resolves this.
+dyanmic_framework_pods += %w[Sentry SentryPrivate]
 pre_install do |installer|
   static = []
   dynamic = []
   installer.pod_targets.each do |pod|
-    # Statically linking Sentry results in a conflict with `NSDictionary.objectAtKeyPath`, but dynamically
-    # linking it resolves this.
-    if %w[Sentry SentryPrivate].include? pod.name
-      dynamic << pod
-      next
+    use_dynamic_frameworks = false
+    if dyanmic_framework_pods.include? pod.name
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/cc13e086c99bb1b5e8c2b87468a6014528febb23/Podfile#L268"></span>Style/IfUnlessModifier: Favor modifier <code>if</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>